components/camel-couchbase/src/main/java/org/apache/camel/component/couchbase/CouchbaseConsumer.java

```diff
@@ -87,7 +87,7 @@ protected void doInit() {
 
         String rangeStartKey = endpoint.getRangeStartKey();
         String rangeEndKey = endpoint.getRangeEndKey();
-        if ("".equals(rangeStartKey) || "".equals(rangeEndKey)) {
+        if (rangeStartKey.isEmpty() || rangeEndKey.isEmpty()) {
             return;
         }
         viewOptions.startKey(rangeEndKey).endKey(rangeEndKey);
```

Comment:
Not related but this sounds suspicious, `rangeStartKey` is not used

Reply:
https://issues.apache.org/jira/browse/CAMEL-19248